plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardRuntimeChangeListener.java

```diff
@@ -107,7 +106,6 @@ public void handleEvent(IFacetedProjectEvent event) {
     }
 
     // Add the App Engine facet
-    IProject project = facetedProject.getProject();
```

Comment:
This is a vestige of refactoring done in commit d337a34335cc2838efb02bf1a4a119a834fb4053. `InstallAppEngineFacetJob` will retrieve the project inside the job.